Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/CheckVersionIntegrityCommand.cs

```diff
@@ -91,17 +91,20 @@ private FileIntegrity CheckFile(AppContentSummaryFile file)
                 return new FileIntegrity(file.Path, FileIntegrityStatus.MissingMetaData);
             }
 
-            if (_localMetaData.GetEntryVersionId(file.Path) != _versionId)
+            int entryVersionId = _localMetaData.GetEntryVersionId(file.Path);
+            if (entryVersionId != _versionId)
             {
-                return new FileIntegrity(file.Path, FileIntegrityStatus.InvalidVersion);
+                string message = string.Format("Expected {0}, but is {1}", _versionId, entryVersionId);
+                return new FileIntegrity(file.Path, FileIntegrityStatus.InvalidVersion, message);
             }
 
             if (_isCheckingSize)
             {
                 long size = new FileInfo(localPath).Length;
                 if (size != file.Size)
                 {
-                    return new FileIntegrity(file.Path, FileIntegrityStatus.InvalidSize);
+                    string message = string.Format("Expected {0}, but is {1}", file.Size, size);
```

Comment:
Please change the name of `size` variable to `actualSize`. I had to look what is assigned to that variable to know what is in it.